src/core/consent/createConsentStateMachine.js

```diff
@@ -14,14 +14,18 @@ import { defer } from "../../utils";
 
 export const DECLINED_CONSENT = "The user declined consent.";
 export const DECLINED_CONSENT_ERROR_CODE = "declinedConsent";
+export const CONSENT_SOURCE_DEFAULT = "default";
+export const CONSENT_SOURCE_INITIAL = "initial";
+export const CONSENT_SOURCE_NEW = "new";
 
-const createDeclinedConsentError = () => {
-  const error = new Error(DECLINED_CONSENT);
+const createDeclinedConsentError = errorMessage => {
+  const error = Error(errorMessage);
```

Comment:
Did you intentionally remove `new`?

Reply:
If this was intentional, let's talk, since we're using `new` everywhere else I believe.

Reply:
Not intentional, I'll fix it.